docs/resources.md

```diff
@@ -1,5 +1,30 @@
 # Development Resources
 
+## Tools:
+- [WebUI-DevKit](https://github.com/dga711/msfs-webui-devkit) - In-game development overlay for html-ui content. Provides fast reload, console output, and more!
+- [devtools-backend-refurb](https://github.com/dga711/msfs-webui-devkit) - Chrome devtools server targeting Coherent GT. Gives you devtools for html-ui content. WARNING: This is still a work in progress, and tends to be very finnicky.
```

Comment:
```suggestion
- [devtools-backend-refurb](https://github.com/dga711/devtools-backend-refurb) - Chrome devtools server targeting Coherent GT. Gives you devtools for html-ui content. WARNING: This is still a work in progress, and tends to be very finnicky.
```
Duplicate link, probably meant this link

Reply:
uhh yeah... whoops!